File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/shader/ShaderTexture.java
package com.abubusoft.xenon.shader;

import com.abubusoft.xenon.R;
 
public class ShaderTexture extends Shader {
	public ShaderTexture() {
		
		builder=ShaderBuilder.build(R.raw.shader_texture_vertex, R.raw.shader_texture_fragment, ArgonShaderOptions.build());
	}
	
}

File: xenon-android-library/src/main/kotlin/com/abubusoft/xenon/shader/ShaderTexture.kt
package com.abubusoft.xenon.shader

import com.abubusoft.xenon.R

class ShaderTexture : Shader() {
    init {
        builder = ShaderBuilder.Companion.build(R.raw.shader_texture_vertex, R.raw.shader_texture_fragment, ArgonShaderOptions.Companion.build())
    }
}